contracts/deploy/sonic/003_sonic_staking_strategy.js

```diff
@@ -98,6 +111,30 @@ module.exports = deployOnSonic(
           signature: "setRegistrator(address)",
           args: [addresses.sonic.validatorRegistrator],
         },
+        // 4. Set 10% performance fee
+        {
+          contract: cOSonicVault,
+          signature: "setTrusteeFeeBps(uint256)",
```

Comment:
I think we need to set the Trusteee Address as well. Will add this in a separate commit

Reply:
done here: https://github.com/OriginProtocol/origin-dollar/pull/2335/commits/f850d728d91edd6b59cdbd2503c593e1b6fa5308